doc/user_guide/howto_create_new_dialect.md

```diff
@@ -0,0 +1,75 @@
+# How to Create a New Dialect of the Document Files Virtual Schema
+
+This guide helps you to create your custom dialect.
+
+## Check if this is the correct guide
+
+First of all, check if you really need a dialect.
+A dialect implements support for an external file provider like BucketFs or S3.
+A dialect does not implement support for different file types see the [user_guide](user_guide.md).
+
+Also, check that you really want a new dialect of the files virtual schema. 
+If your data source is not a file storage, but for example a database, 
+you probably do not want to create a dialect of this Virtual Schema,
+but of the [virtual-schema-common-document](https://github.com/exasol/virtual-schema-common-document).
+
+ ## Getting started
+ 
+ To get started with a new dialect create a new repository and copy the contents of the [bucketfs dialect](https://github.com/exasol/bucketfs-document-files-virtual-schema).
+ 
+ ## Adapt the pom.xml file
+ 
+ Now go to the `pom.xml` file and replace all occurrences of `bucketfs` by your file provider.
+ If you are not part of the Exasol company, also don't forget to change the organization.
+ 
+ ## `AdapterFactory`
+ 
+ The entry point for each virtual schema is the AdapterFactory. 
+ It is loaded via a service loader and build the virtual schema adapter.
+ For BucketFS it is the `BucketfsDocumentFilesAdapterFactory`.
+ Rename this class to `YOUR_FSDocumentFilesAdapterFactory` and also replace `bucketfs` in `getAdapterVersion()`.
```

Comment:
Don't ask to copy and rename. Check how we described the creation of JDBC dialects